content/basics/vimmodes/visual.md

```diff
@@ -4,4 +4,14 @@ date =  2017-10-30T10:49:01-04:00
 weight = 7
 +++
 
-Visual Mode
+Visual Mode is where you can select words,lines,blocks,etc... There are some keys to enter in this mode, let's se them:  
```

Comment:
Visual Mode is where you can select words, lines, blocks, etc... While in **Normal Mode** you can use the following keys to switch to this mode: